### DISCUSSION TITLE
Is this project still actively maintained?

### DISCUSSION THREAD

**Discussion by GustavoOS** [standard/ts-standard#294] [Q&A]
With new TS versions released, it seems the project is no longer being actively maintained.
Last version on npm is from 2022 (over a year ago).

Can anyone confirm that, please?

**Comment by ridomin** [comment]
no response since March, no PRs merged since 2023

.. and not targeting last version of TS.

not sure if it's worth to try to contribute with a PR

**Comment by GustavoOS** [comment]
@theoludwig would you mind deprecating this package?

**Reply by theoludwig** [reply]
I can't deprecate it as I don't have the permissions on npm.

But I agree that this package should be deprecated, the same thing for the whole Standard organization and all the related packages.

Might be replaced by https://github.com/neostandard/neostandard, maintained by @voxpelli.

**Comment by voxpelli** [comment]
See https://github.com/standard/standard/issues/1948#issuecomment-2138078249
